code/Test_definitions/qos-profiles-retrieveQoSProfiles.feature

```diff
@@ -0,0 +1,224 @@
+Feature: CAMARA QoS Profiles API, v0.11.0 - Operation retrieveQoSProfiles
+    # Input to be provided by the implementation to the tester
+    #
+    # Implementation indications:
+    # * apiRoot: API root of the server URL
+    # * List of device identifier types which are not supported, among: phoneNumber, ipv4Address, ipv6Address.
+    #   For this version, CAMARA does not allow the use of networkAccessIdentifier, so it is considered by default as not supported.
+    #
+    # Testing assets:
+    # * The name of an existing QoS profile
+    # * If some QoS Profile is restricted for some devices, provide the QoS profile name and device
+    # * A device object identifying a device commercialized by the implementation for which the service is not applicable, if any
+
+
+    # References to OAS spec schemas refer to schemas specifies in qos-profiles.yaml, version 0.11.0-rc.1
```

Comment:
Here `-rc.1` is wrong as the renaming to `retrieve-qos-profiles` happened after -rc.1 :-)

 ```suggestion
    # References to OAS spec schemas refer to schemas specifies in qos-profiles.yaml, version 0.11.0
```